plugin.video.hdhomerun.simple/resources/lib/hdhomerun.py

```diff
@@ -523,7 +523,7 @@ def resolveURL(self, tunerkey, channel, opt=None):
         url   = info.getURL()
         tuner = info.getTuner()
         if tuner.getModelNumber() == "HDTC-2US":
-            if self.transcode == 'none': tranOPT = (tuner.getTranscodeOption() or 'none')
+            if self.transcode == 'default': tranOPT = (tuner.getTranscodeOption() or 'none')
```

Comment:
Thanks, I already made a similar change... HDHR API docs indicate no 'default'; I replace it altogether with 'none'.

Reply:
From what I can tell 'default' is ok/valid in your hdhomerun.py code - at least it is now. When hdhomerun.py sees that 'default' is the selected hdhomerun simple transcode configuration it will query the tuner itself and see what has been set as the default for that tuner.   The problem is that is what was not working in pyHDR.py and messed things up.  Now that pyHDHR.py is working 'default' will be replaced with the setting on the tuner itself (ie with none, mobile, heavy, etc)